Gemfile

```diff
@@ -62,8 +62,8 @@ gem 'tumblr_client', github: 'albertsun/tumblr_client', branch: 'master', ref: '
 gem 'omniauth-tumblr', '~> 1.2'
 
 # Dropbox Agents
-gem 'dropbox-api'
-gem 'omniauth-dropbox'
+gem 'dropbox-api', github: 'dsander/dropbox-api', ref: '86cb7b5a1254dc5b054de7263835713c4c1018c7'
```

Comment:
What are the pending PRs we should keep tracking?

Reply:
https://github.com/omenking/dropbox-api/pull/1 It's a PR to a fork of the original gem which already included most of the changes needed for V2 of the API. The original seems unmaintained so we either have to release a new `dropbox-api2` gem ourselves or rely on the GitHub fork.